src/os.rs

```diff
@@ -19,55 +19,46 @@ use std::io::Read;
 
 use rand_core::{RngCore, Error, ErrorKind, impls};
 
-/// A random number generator that retrieves randomness straight from
-/// the operating system.
+/// A random number generator that retrieves randomness straight from the
+/// operating system. This is the preferred external source of randomness for
+/// most applications. Commonly it is used to initialize a user-space RNG, which
+/// can then be used with much lower overhead.
 ///
-/// Platform sources:
+/// Sometimes there is the misconception that `OsRng` somehow provides 'better'
+/// random numbers than some good user-space cryptographic random number
+/// generator. This is not true, but `OsRng` has a role in initializing such an
```

Comment:
This is a very incomplete note on the topic; it would be better just to refer to elsewhere (e.g. the doc I wrote, or maybe we should refer to an external article on the topic specific to our Rand lib).

Reply:
My goal was mostly to say that CSPRNG's generate values that can be just as good as those from `OsRng`, but I can see what you mean. There may be other considerations. I did a second try at this paragraph.

If it is not yet really what you have in mind, can you make a suggestion?